priv/test_files/with_warnings/src/a_module.erl

```diff
@@ -1 +1,15 @@
 -module(a_module).
+
+-export([my_fun/2, multi_fun/2]).
+
+%% _Arg2 is unused
+my_fun(Arg1, _Arg2) ->
```

Comment:
We need another test function where `_` is used instead of `_Arg2`. In `erl_syntax` lingo, `_` is an `underscore` while `_Arg2` is a variable.
We also need a test function where the same argument is once ignored with `_Arg2` and then with `_`.
And also a test function where multiple arguments are ignored in all clauses.